# Welcome to the Van Galen Lab Github page

![banner](https://github.com/vangalenlab/.github/blob/f59c9f6d6234357b346c99a9b00ce3d77f9fb2f3/profile/banner.jpg)

Here we highlight software that was developed by Van Galen Lab members and collaborators.

## Project: MAESTER
Accompanying paper: Miller et al, Mitochondrial variant enrichment from high-throughput single-cell RNA-seq resolves clonal populations, [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.03.08.434450v1)

### [maegatk - Mitochondrial Alteration Enrichment and Genome Analysis Toolkit](https://github.com/caleblareau/maegatk)
Developed by Caleb Lareau, the maegatk package processes `.bam` files with mitochondrial reads to generate high-quality heteroplasmy estimation from sequencing data.

### [Mitochondrial variants](https://github.com/EDePasquale/Mitochondrial_variants)
We developed a table of mitochondrial variants, their effects on the resultant proteins, and the potential functional consequences to assess the potential beneficial/detrimental consequences of mutations.

### [MAESTER-2021](https://github.com/vangalenlab/MAESTER-2021)
Various scripts used to analyze the data and generate figures for the paper.

## Project: BPDCN
Accompanying paper: DePasquale et al, Single-cell Multiomics Reveals Clonal T-cell Expansions and Exhaustion in Blastic Plasmacytoid Dendritic Cell Neoplasm, [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.12.01.470599v1)

### [WARPT - Workflow for Association of Receptor Pairs from TREK-seq](https://github.com/mainciburu/WARPT)
This is a pipeline to process data generated by the protocol T-cell Receptor Enrichment to linK clonotypes by sequencing (TREK-seq).

### [BPDCN](https://github.com/EDePasquale/BPDCN)
Various scripts used to analyze the data and generate figures for the paper.

